amazon-redshift-plugin/src/main/java/io/cdap/plugin/amazon/redshift/RedshiftSchemaReader.java

```diff
@@ -114,4 +92,66 @@ public List<Schema.Field> getSchemaFields(ResultSet resultSet) throws SQLExcepti
     return schemaFields;
   }
 
+  // schema function updation
+
+  /**
+   * Override: Fetches schema fields for a specific table using database metadata.
+   */
+  @Override
```

Comment:
It is there in parent class, you can use that one, instead you can call getSchema method of this class inside

Reply:
update javadoc